packages/components/src/components/post-icon/post-icon.scss

```
@@ -1,9 +1,103 @@
 :host {
-  display: block;
+  display: inline-block;
+  width: 1em;
+  vertical-align: -0.15em;
```

Is `middle` not the value we want here?

```suggestion
  vertical-align: middle;
```

---

I tested with `middle` and some other values. And the answer is "no". The value `-0.15em` (which I found in other icon systems) fits best.